robomimic/envs/env_robosuite.py

```diff
@@ -136,14 +137,10 @@ def reset_to(self, state):
         should_ret = False
         if "model" in state:
             self.reset()
-            robosuite_version_id = int(robosuite.__version__.split(".")[1])
-            if robosuite_version_id <= 3:
-                from robosuite.utils.mjcf_utils import postprocess_model_xml
-                xml = postprocess_model_xml(state["model"])
-            else:
-                # v1.4 and above use the class-based edit_model_xml function
-                xml = self.env.edit_model_xml(state["model"])
-            self.env.reset_from_xml_string(xml)
+            # ----- loading LIBERO model xml ----
```

Comment:
What was this change about btw?

Reply:
This is not a change I made intentionally, might be due to a version mismatch. Reverting